src/xo-app/vm/tab-advanced.js

```diff
@@ -294,6 +295,16 @@ export default connectStore(() => {
   <Container>
     <Row>
       <Col className='text-xs-right'>
+        {isAdmin &&
```

Comment:
Shouldn't the self user(s) also be able to share the VM? @olivierlambert?

Reply:
indeed, if it's the current VM "owner", it should be able to "share" it